### DISCUSSION TITLE
Unable to share my project to Github with Android Studio

### DISCUSSION THREAD

**Discussion by Uzair-Jamal** [community/community#130041] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I have tried everything that I was doing in the previous Android Studio version, but somehow I am not able to share my project on GitHub with Android Studio. Kindly suggest me ways to solve this issue, for now I am stuck and not able to work properly.

I tried through steps(VCS > GitHub > Share Your project on github), but that was not working for me also I tried with steps (settings > Version Control > GitHub) sometimes it shows (bad credentials) but in my case it is "OK" everything is fine. If it is "Bad Credentials" then you have to re-login through your password. Now I am stuck in a situation where I am unable to share my work on Github. I should have the respective laptop to work on the project. Kindly do help me!
![image](https://github.com/community/community/assets/85803617/b4bf5ade-770a-4afe-8d0f-b5832ab53784)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by o-david** [comment]
I strongly recommend using your terminal/command prompt. You can also use my simplified CLI https://www.npmjs.com/package/easygit-cli

**Comment by opeyemi-osoba** [comment]
I actually have exactly thesame issue too presently. The function works perfectly on IntelliJ Idea though. So I think it is an Android Studio issue. However, I created a Ticket on GitHub, will update you about the outcome.

**Comment by Uzair-Jamal** [comment]
Thanks alot

**Comment by opeyemi-osoba** [comment]
Hi Jamal,
as promised, find below the response from GitHub support.

*****START*****

Thanks for getting in touch with GitHub Support. My name is Tempy and I am happy to assist you here!
 
JetBrains has discovered a new security issue that affects the JetBrains GitHub plugin on the IntelliJ Platform, which could lead to disclosure of access tokens to third-party sites, e.g. GitHub. The issue affects all IntelliJ-based IDEs as of 2023.1 onwards that have the JetBrains GitHub plugin enabled and configured/in-use.
 
The issue is now resolved and a fix has been provided for all IDEs based on the IntelliJ Platform from version 2023.1 onwards.
 
JetBrains has release a solution for all users and you can find necessary steps [in JetBrains blogpost here](https://blog.jetbrains.com/security/2024/06/updates-for-security-issue-affecting-intellij-based-ides-2023-1-and-github-plugin/#what-is-required-of-you).
 
Let me know if there is anything else I can help you with!
 
With kind regards,
*****END*****

**Comment by Uzair-Jamal** [comment]
Thank you for your help!
